cloud-controller-manager/cmd/digitalocean-webhook-server/main.go

```diff
@@ -40,25 +100,29 @@ func main() {
 }
 
 func startWebhookServer() error {
-	// default server running at port 9443, looking for tls.crt, tls.key in /tmp/k8s-webhook-server/serving-certs
-	ll := ctrl.Log.WithName("webhook-validation-server")
-	ll.Info("getting config")
-	config, err := ctrl.GetConfig()
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "failed to retrieve kubeconfig %v\n", err)
-	}
+
+	ll := zap.New(logOpts...).WithName("validation-webhook-server")
+	ctrlruntimelog.SetLogger(ll)
+	ll.Info("THIS  VALIDATION WEBHOOK SERVER IS STILL IN DEVELOPMENT AND IS PURELY EXPERIMENTAL")
+
 	server := webhook.Server{}
-	c, err := client.New(config, client.Options{Scheme: scheme})
+
+	DOClient, err := initDOClient()
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "failed to construct cr client %v\n", err)
+		return fmt.Errorf("failed to initialize DO client: %s", err)
 	}
 
-	server.Register("/validate-doks-lb-service", &webhook.Admission{Handler: &do.DOKSLBServiceValidator{Client: c, Log: ll}})
-	ll.Info("Registering Webhook server handlers")
-	if err = server.StartStandalone(ctrl.SetupSignalHandler(), scheme); err != nil {
-		return err
+	server.Register("/validate-doks-lb-service", &webhook.Admission{Handler: &do.KubernetesLBServiceValidator{
+		Log:     ll,
+		GClient: DOClient,
+	}})
+
+	ll.Info("Registering loadbalancer validation webhook server handlers")
+	if err := server.StartStandalone(ctrl.SetupSignalHandler(), scheme); err != nil {
+		ll.Error(err, "failed to start loadbalancer validation webhook server")
+		os.Exit(1)
 	}
-	ll.Info("Webhooks server started")
+	ll.Info("loadbalancer validation webhook server started")
```

Comment:
[AFAICS](https://github.com/kubernetes-sigs/controller-runtime/blob/cd65cb25d314f40a329a688f4714fe3282589e97/pkg/webhook/server.go#L291C16-L293), `StartStandalone` only returns when the server terminates. it also logs graceful shutdowns and errors, meaning we can and should drop this log line.